demo/03-create-cluster.sh

```diff
@@ -216,7 +217,8 @@ main() {
       .properties.platform.subnetId = $subnet_id |
       .properties.platform.networkSecurityGroupId = $nsg_id |
       .properties.platform.operatorsAuthentication.userAssignedIdentities = $uamis_json_map |
-      .identity.userAssignedIdentities = $identity_uamis_json_map
+      .identity.userAssignedIdentities = $identity_uamis_json_map |
+      .properties.spec.network.networkType = $network_type
```

Comment:
This property doesn't exist anymore.  For the API review for MSFT we removed any field where the feature wasn't complete (or close to complete) so I don't know if this PR is valid anymore from frontend perspective.